deps/spidershim/test/value.cc

```diff
@@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
   Local<Float64Array> f64a =
       CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
 }
+
+static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
+  v8::Local<v8::Script> result;
+  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
+          .ToLocal(&result)) {
+    return result;
+  }
+  return v8::Local<v8::Script>();
+}
+
+static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
```

Comment:
On Wed, May 04, 2016 at 10:40:48PM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/test/value.cc:
> 
> > @@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
> >    Local<Float64Array> f64a =
> >        CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
> >  }
> > +
> > +static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
> > +  v8::Local<v8::Script> result;
> > +  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
> > +          .ToLocal(&result)) {
> > +    return result;
> > +  }
> > +  return v8::Local<v8::Script>();
> > +}
> > +
> > +static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
> 
> Can you instead please use engine.CompileRun like other tests do?

I could try I guess, but is there really value in diverging from the v8
tests?  I think there may well be more v8 tests using these helpers, and
I don't really see a reason to make stealing them harder.